articles/050_ros_rpc_design.md

```diff
@@ -92,16 +92,16 @@ For logging/introspection purposes the RPC Server instance might publish all inc
 
 ### ROS Preemptible RPC API
 
-The ROS preemptible RPC API will extend the Asynchronous API to enable preemption of an RPC in progress using a UID.
+The ROS preemptible RPC API will extend the Asynchronous API to enable preemption of an RPC in progress using a user identifier (UID).
```

Comment:
That's a good question. I left it as `UID` because it could be `GUID` if the request is uniquely identified by the requester plus a sequence number, but otherwise you might need a full `UUID` if the id must be unique across all service requests.